templates/security/fips.html

```diff
@@ -213,15 +218,15 @@ <h2>Certified packages under FIPS 140</h2>
       </tbody>
     </table>
     <a class="p-button--neutral js-invoke-modal" href="/security/certifications/docs/fips">Read more about the FIPS certification</a>
-    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140 validated modules</a>
+    <a class="p-button--positive js-invoke-modal" href="/advantage">Access the FIPS140 validated modules</a>
```

Comment:
Can you remove the `js-invoke-modal` class, as this will trigger the previous contact form. Thanks

Reply:
done